README.md

```diff
@@ -110,6 +110,7 @@
 * [nix-init](https://github.com/nix-community/nix-init) - Generate Nix packages from URLs with hash prefetching, dependency inference, license detection, and more.
 * [nix-melt](https://github.com/nix-community/nix-melt) - A ranger-like flake.lock viewer.
 * [nix-prefetch](https://github.com/msteen/nix-prefetch) - A universal tool for updating source checksums.
+* [nix-output-monitor](https://git.maralorn.de/nix-output-monitor/) - A tool to get additional information while building.
```

Comment:
```suggestion
* [nix-output-monitor](https://git.maralorn.de/nix-output-monitor/) - A tool to produce useful graphs and statistics when building derivations.
```

The original description is correct, both factually and grammatically, but kinda undersells the tool, so I'd recommend the above word choice, but that's up to you.